imod_coupler/drivers/dfm_metamod/mapping.py

```diff
@@ -246,10 +248,11 @@ def mapping_passive_mf_dflow1d(
             ndmin=2,
             skiprows=1,
         )
+        ptx = table_passive_mfriv2dflow1d[:, 0]
```

Comment:
ptx anbd pty are alro used below as variable name, which can be confusing. How about ptx_riv and pty_riv/ ptx_drain and pty_drain?